src/services/Store/store/settings.slice.ts

```diff
@@ -40,6 +40,9 @@ const slice = createSlice({
     },
     setInactivityTimer(state, action: PayloadAction<number>) {
       state.inactivityTimer = action.payload;
+    },
+    toggleDemoMode(state, action: PayloadAction<boolean>) {
```

Comment:
It isn't really a toggle if you provide the boolean value is it? 😄  Would prefer `setDemoMode` or something similar.